defaults/main.yml

```
@@ -2,6 +2,7 @@
 postgresql_backup:
   enabled: false
   directory: ''
+  cron: '30 03 * * *'
```

@alexlanz 
do I need the quotes here?

---

```suggestion
  time: '30 03 * * *'
```

You don't need the quotes, but is also no problem if you leave them.

What do you think about giving the property a more specific name?

---

I think `cron` is actually more explicit then `time`. 

For example in Spring boot you can schedule jobs and they also name it cron:

`@Scheduled(cron = "30 03 * * *")`

---

Ok, works for me